src/main/java/com/snack/news/dto/AdminNewsDto.java

```diff
@@ -0,0 +1,53 @@
+package com.snack.news.dto;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.snack.news.domain.category.Category;
+import com.snack.news.domain.news.News;
+import com.snack.news.domain.tag.Tag;
+import com.snack.news.domain.topic.Topic;
+import com.snack.news.domain.topic.TopicType;
+import lombok.*;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid unused imports such as 'lombok'](https://app.codacy.com/app/snack-news/Snack-BE/pullRequest?prid=4677893)